source/_components/suez.markdown

```diff
@@ -0,0 +1,59 @@
+---
+layout: page
+title: "Suez Water Sensor"
+description: "Instructions on how to integrate Suez Water daily data within Home Assistant."
+date: 2019-06-06 11:11
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: suez.png
+ha_release: 0.95
+ha_category:
+  - Water
```

Comment:
Remove this. It would create an additional category with one entry.

Reply:
Done